birdy/native/__init__.py

```diff
@@ -0,0 +1,49 @@
+"""
+The :func:`import_wps` function *imports* on the fly a python module whose
```

Comment:
Update the doc to reflect that the object created is a class with methods calling processes.